force-app/main/default/permissionsets/Feature_Community_User.permissionset-meta.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<PermissionSet xmlns="http://soap.sforce.com/2006/04/metadata">
```

Comment:
this perm set is redundant, should be removed. since the Feature A and B perm sets already exist. Those can directly be assigned to the exp cloud users who get the exp cloud license assigned. Even though it is two separate perm sets, this would help illustrate that not every entitled value gives access to the feature until each one is granted.